mumbleBot.py

```diff
@@ -732,14 +735,32 @@ def start_web_interface(addr, port):
     logging.getLogger("root").addHandler(handler)
     var.bot_logger = bot_logger
 
+
+    # ======================
+    #     Load Database
+    # ======================
+    var.settings_db_path = args.db if args.db is not None else util.solve_filepath(
+        config.get("bot", "database_path", fallback="settings.db"))
```

Comment:
default should be `settings-<name of the bot>.db` . This allow multiple bot without naming manually the db.

Reply:
Will do